inc/cache.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+class SiteOrigin_Panels_Cache {
```

Comment:
Yeah, agreed. Have renamed it to `SiteOrigin_Panels_Cache_Renderer`. I'll look a little bit more at making things more obvious.